Document funding in pubspec.yaml (#4190)
dart-lang/site-www: src/tools/pub/pubspec.md

{% endprettify %}

### Funding

> Package authors may use the `funding` property specify a list of URLs that
> provide information on how users can help fund the development of the package.

Reviewer: ```suggestion
Package authors can use the `funding` property to specify a list of URLs that
provide information on how users can help fund the development of the package.
For example:
```

> {% prettify yaml tag=pre+code %}

Reviewer: Hmmm. Is dart.dev still using prettify?

Reviewer 2: Only in cases where we need to highlight code, so it's fine to switch this case to a normal markdown code block.

funding:
 - https://www.buymeacoffee.com/example_user
 - https://www.patreon.com/some-account
{% endprettify %}

> If published to [pub.dev]({{site.pub}}) the links will be displayed on the
> package page. This aims to help users that wish to help fund the development
> of their dependencies.

Reviewer: ```suggestion
If published to [pub.dev]({{site.pub}}) the links are displayed on the
package page. This aims to help users fund the development
of their dependencies.
```

{% endprettify %}

### Funding

Package authors can use the `funding` property to specify a list of URLs that
provide information on how users can help fund the development of the package.
For example:

> {% prettify yaml tag=pre+code %}
> funding:
>  - https://www.buymeacoffee.com/example_user
>  - https://www.patreon.com/some-account
> {% endprettify %}

Reviewer 2: ````suggestion
```yaml
funding:
 - https://www.buymeacoffee.com/example_user
 - https://www.patreon.com/some-account
```
````
